### PR TITLE
Move to IT

### PR DESCRIPTION
**Pull request by The-smooth-operator**

### CODE REVIEW

**Comment by The-smooth-operator** [comment]
@johngian @akatsoulas can you please review these changes?
The current version deployed in the IT infra has already those changes implemented

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,37 @@
...
+  - secure: ol+plznHz48j8hb0vqh764e3lEGD19glQFsv6ZTy1VPUviaFEODNbt6xhs/m8wI/qU/xTp9w8RVpMoYyYKcJ+W6a8AX7F/4oWavBqxwXYk5imrBKjxQ5R6JYl3LsNRhxoqRZCfIZad7zdfXzSj5mJVb9nO15Daiq7qIZjpfx9uKT37PtWMRVRzsLplgODlHHFFAo3WY2ZOD3BK0DOfNIsxKa3of6n6ALNpLNSgOuCtFsBmvIKN1VY9drWaDpepLUczs8vLJlua8nB1RGVYF+bgOukZJGTyAt5OeROpeXUs1w7rdUxMk5JgkeqGhoD2z1XC4EH/h8km7/QFaP7gy9YR4DupfSUZWev+7W136G2XMQUPJ8b1zy++npj4vps9liUv3G0MO+ASX1pcsedKTJSyUAs0DEAYkpZfAGkquA4IrKlkOivQ6T3kyOqtmPXVNdVSquUpEG2oFamWL7qyJ+YqufegSyTw2bkmSVEGhWUBJsPqyYZVAN9PQJZE3LR91iJnuao6EbAfmLtWwK2qqkL4qVxAhL2MvyM9vcR062HkFuGld3Tt5UssxIbf4RXUcRMnACiycS0h9b81beRY/Ttab1ewV/c51PlIqam+UAXqHW/ez4a/2g6mfbF6/oO/i2H2mbSD9cslZGEcpLg3x6fONf0pSbL8xgG0mfZPw8xos=
+  - ECR="$AWS_ACCOUNT_ID.dkr.ecr.us-west-2.amazonaws.com/moderator"
```
Comment (johngian) [inline .travis.yml]: Let's add the whole ECR URL to an env var instead.

**Comment by johngian** [comment]
Hey @The-smooth-operator ! I added a nit comment. Overall it looks OK and you've already been deploying this on stage using your fork so it should work fine.

r+

**Comment by johngian** [comment]
It looks like travis needs some configuration to work in this repository build.

**Comment by The-smooth-operator** [comment]
FYI: the reason why Travis is failing is because I have to re-encrypt the secrets fro this repo.

**Pull request closed without merging**